src/main/java/com/github/dikhan/domain/TriggerIncident.java

```diff
@@ -1,67 +1,47 @@
 package com.github.dikhan.domain;
 
-import java.util.List;
-import java.util.Objects;
-
 import com.fasterxml.jackson.annotation.JsonProperty;
 
-public class TriggerIncident extends Incident {
+import java.util.Objects;
 
+public class TriggerIncident extends Incident {
     @JsonProperty("client")
-    private final String client;
+    private String client;
```

Comment:
Since the attribute value is expected to be immutable it is preferable to make it final to make it clear. Not like it will affect anything if we don't in this case as there are no setters and the type is String, but it's a good practise to define constructor values as un-mutable and therefore final so consumers of the class can not mutate the state of the object at any given time.